contracts/core/VolOracle.sol

```diff
@@ -98,24 +113,35 @@ contract VolOracle is DSMath {
             "Committed"
         );
 
-        (uint256 newCount, int256 newMean, uint256 newM2) =
-            Welford.update(accum.count, accum.mean, accum.m2, logReturn);
+        require(observations[pool].length > 0, "!pool initialize");
```

Comment:
We can move this to the top of the function because this will be the first thing to revert if the pool is not initialized